basic.js

```diff
@@ -523,13 +523,15 @@ this.basic = (function() {
       },
 
       'on_goto': function ON_GOTO(index /* , ...lines */) {
+        if (index < 0 || index > 255) {
```

Comment:
This doesn't handle index === 255.1
I think we want `index = Math.floor(index)`